governance.md

```diff
@@ -129,6 +130,21 @@ committee has a charter and a chair, and will report to the steering
 committee periodically, and to the community as makes sense, given the
 charter.
 
+## User groups
+Some topics have long term relevance to large groups of Kubernetes users, but
+do not have clear deliverables or ownership of parts of the Kubernetes
+code base. As such they are neither good fits for SIGs or Working Groups.
+Examples of such a topic might be continuous delivery to Kubernetes.
+
+User groups function as a centralized resource to facilitate communication
+and discovery of information related to the topic of the user group. User
+groups should not undertake to produce any deliverable, instead they should
+form working groups under the auspices of some SIG for such work. Likewise
+they shouldn't take ownership of anything in the Kubernetes process, as
+that is a role for SIGs.
+
+User groups are documented in [sigs.yaml](sigs.yaml)
```

Comment:
in order to facilitate discoverability and engagement

Reply:
done.